File: lecture_notes/08 collections/2-function.py
# Implementation

def getFirstAndLast(ages):
  first = ages[0]
  last = ages[-1]
  return first, last

# Usage

ages = [10, 12, 15, 15, 17, 18, 18, 19, 20]
first, last = getFirstAndLast(ages)
print({ 'first': first, 'last': last })

File: lecture_notes/08 collections/2-function.js
'use strict';

// Implementation

const getFirstAndLast = (array) => {
  const first = array[0];
  const last = array[array.length - 1];
  return { first, last };
};

// Usage

const ages = [10, 12, 15, 15, 17, 18, 18, 19, 20];
const { first, last } = getFirstAndLast(ages);
console.log({ first, last });
